Python.gitignore

```diff
@@ -44,6 +44,7 @@ nosetests.xml
 coverage.xml
 *,cover
 .hypothesis/
+!.coveragerc
```

Comment:
I *think* this is being caught by `.coverage`, right?

If so, can we move this up to there and add a comment explaining why it needs to exist...